geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -201,8 +201,11 @@ object RichSimpleFeatureType {
   // in general we store everything as strings so that it's easy to pass to accumulo iterators
   implicit class RichSimpleFeatureType(val sft: SimpleFeatureType) extends AnyVal {
 
-    def getGeomField: String = sft.getGeometryDescriptor.getLocalName
-    def getGeomIndex: Int = sft.indexOf(sft.getGeometryDescriptor.getLocalName)
+    def getGeomField: String = {
```

Comment:
should these be Option[String] and Option[Int] like getDtgField is for consistency?